go-controller/pkg/ovn/endpoints.go

```diff
@@ -192,7 +215,10 @@ func (ovn *Controller) deleteEndpoints(ep *kapi.Endpoints) error {
 			}
 			// ClusterIP may be on gateway or worker LBs, so need to remove here as well
 			if config.Gateway.Mode == config.GatewayModeShared {
-				ovn.clearVIPsAddRejectACL(svc, gatewayLB, svc.Spec.ClusterIP, svcPort.Port, svcPort.Protocol)
+				err = ovn.configureLoadBalancer(gatewayLB, svc.Spec.ClusterIP, svcPort.Port, nil)
```

Comment:
how are you handling moving the VIP off of lets say an idle load balancer to a cluster load balancer for an update? I dont see you clearing the VIP on the other one (kind of like how I used to clear the vip here)

Reply:
The new logic is in line 189:
https://github.com/ovn-org/ovn-kubernetes/pull/2124/files/ed040a146f96c03330114bb921f2e2566f3ff706..47e6360fa5a60fcc00c8201bbcfa4837897c9dca#diff-26c6f830f04d673799520de55b6a185d489b8b32852d6673d10a902ac5803ed3R189

If the service needs idling, we remove from loadbalancers (all of them) and add to idling, otherwise we remove from idling and let the current logic flow.